packages/k2-alpine/App.tsx

```diff
@@ -25,5 +28,9 @@ export default function RootLayout(): React.JSX.Element | null {
     return null
   }
 
-  return <StorybookUI />
+  return (
+    <K2AlpineThemeProvider colorScheme={colorScheme}>
```

Comment:
can we move colorScheme inside the provider?

Reply:
fixed. but colorScheme prop(optional) will be still there because we should override it to dark for the current app.